src/tests/multihost/docs/Install.rst

```diff
@@ -17,8 +17,9 @@ Fedora 35
 
 * Default branch when sssd is cloned is the master branch::
 
-   $ [testuser@dhcp201-228 sssd]$ git branch
-   * master
+    $ cd sssd
```

Comment:
Inconsistent with bottom line, `[testuser@dhcp201-228 ~]` is missing. Also I am wondering if for simplicity not replace `[testuser@dhcp201-228]` with some generic `[user@host]` or just skip this part of PS.

Reply:
You are right. I fixed it.

I also modified the prompt as you proposed to "user@host" and made them coherent. Also corrected the current directory displayed in the prompt which wasn't always coherent with the actions done or to be done.